### PR TITLE
DM-43534: Extend confluence_url characters size limit

### PR DESCRIPTION
**Pull request by sebastian-aranda**

### CODE REVIEW

File: alembic/versions/3f4e119594fe_extend_confluence_url_length.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# type: ignore
+# flake8: noqa
+"""extend confluence_url length
```
Comment (wvreeven) [inline alembic/versions/3f4e119594fe_extend_confluence_url_length.py:3]: `extend` -> `Extend`

File: alembic/versions/3f4e119594fe_extend_confluence_url_length.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+Revision ID: 3f4e119594fe
+Revises: e6dc052f0caf
+Create Date: 2024-04-02 11:01:50.116786
+
+"""
+
+import logging
+
+import sqlalchemy as sa
+
```
Comment (wvreeven) [inline alembic/versions/3f4e119594fe_extend_confluence_url_length.py:14]: I think this empty line may be deleted. Are you using `isort`?

**Review by wvreeven — APPROVED** [review]
Looks good to me. I left a few comments for your consideration.

File: alembic/versions/3f4e119594fe_extend_confluence_url_length.py (hunk 1 above)
Comment (sebastian-aranda) [reply alembic/versions/3f4e119594fe_extend_confluence_url_length.py:3]: I thinks using lower case is kind of a "standard" when using `alembic` https://alembic.sqlalchemy.org/en/latest/tutorial.html#create-a-migration-script. These titles are created using:
`alembic revision -m "title"`, so the usualy are set in lower case. This happens for the rest of the migrations too :wink:.

File: alembic/versions/3f4e119594fe_extend_confluence_url_length.py (hunk 2 above)
Comment (sebastian-aranda) [reply alembic/versions/3f4e119594fe_extend_confluence_url_length.py:14]: We are using `isor` here indeed. I just tried locally making a change so Isort claims about it, and on its formatting it sets the imports as they are now. So I think we are good :+1: (I'm using `ts-pre-commit-config 0.9.5` btw).

**Pull request merged**